package/lib/compileFunctions.js

```diff
@@ -57,6 +58,17 @@ module.exports = {
         });
       }
 
+      if (funcObject.vpcEgress) {
+          let egress = _.get(funcObject, 'vpcEgress') || _.get(this, 'serverless.service.provider.vpcEgress');
```

Comment:
What is the purpose of this line here? Based on `if` from line above, we know that `funcObject` has `vpcEgress` set so we can just use it

Reply:
updated, thanks!